extras/kms/migrations/postgres/04_keys.up.sql

```diff
@@ -56,7 +56,7 @@ create table kms_data_key (
     references kms_root_key(private_id)
     on delete cascade
     on update cascade,
-  purpose text not null unique
+  purpose text not null
```

Comment:
Why is this change needed here?

Reply:
in the course of testing the new cache bits I attempted to add a `database` DEK to two scopes and it failed which is a defect.

Reply:
Ah, I see, a few lines below we have `unique (root_key_id, purpose) -- there can only be one dek for a specific purpose per root key` which is the correct constraint here.